protocols/peralta_yahya_lab/flow_cytometry_prep.ot2.py

```diff
@@ -0,0 +1,53 @@
+from opentrons import labware, instruments
+
+tiprack_1 = labware.load('tiprack-200ul', 7)
+tiprack_2 = labware.load('tiprack-200ul', 8)
+tipracks = [tiprack_1, tiprack_2]
+
+trough = labware.load('tube-rack-15_50ml', 9)
+
+P1 = labware.load('96-flat', 1)  # Chemicals
+P2 = labware.load('96-flat', 2)  # Plate taking in chemical CP1
+P3 = labware.load('96-flat', 3)  # Samples
+P4 = labware.load('96-flat', 4)  # Plate taking in chemical CP2
+P5 = labware.load('96-flat', 5)  # Plate taking in checmial CP3
+P6 = labware.load('96-flat', 6)  # Plate taking in chemical CP4
+
+m50 = instruments.P50_Multi(
+    mount='left',
+    tip_racks=tipracks)
+
+p300 = instruments.P300_Single(
+    mount='right',
+    tip_racks=tipracks)
+
+targets = [P2, P4, P5, P6]
+
+p300.pick_up_tip()
+for target_plate in targets:
+    for col_num in range(1, 11):
+        p300.transfer(182, trough['A1'], target_plate.cols(col_num)[1:7],
+                      new_tip='never')
+p300.drop_tip()
+
+target_list = []
+for col_select in range(2, 5):
+    target_list.append(['B' + str(col_select), 'B' + str(col_select + 3),
```

Comment:
Instead of doing a for loop you could do the following:
plate_info = plate.wells('B2', to='B6', step=2) where it would go to 'B2', skip to 'B4' and finish at 'B6'
or simply make a list of wells via: plate.wells('B2', 'B4', 'B6')
target_list = [plate_info1, plate_info2, plate_info3, plate_info4]

You can then make a list called 'target list' of the wells you need to go to in the given order.